src/ObjectUtils/PdbFileTest.cpp

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
It's a bit unclear what is tested by this due to the filename choice. In addition, on Windows, this is testing PdbFileDia, on Linux this is testing PdbFileLLVM. However, on Windows PdbFileLLVM also exists, but is not tested at all. 

Maybe it would be cleaner to have PdbFileDiaTest and PdbFileLLVMTest separate, with the DIA variant obviously only built and tested on Windows?

Reply:
Agreed, added type-parameterized tests (PdbFileTest.h, PdbFileDiaTest.cpp, PdbFileLlvmTest.cpp). The llvm variant is now tested on Windows.